src/features/rewards/welcomePage/index.tsx

```diff
@@ -84,9 +84,16 @@ class WelcomePage extends React.PureComponent<Props, {}> {
     this.props.optInAction()
   }
 
+  isTouchscreen = () => {
+    return 'ontouchstart' in document.documentElement
```

Comment:
this doesn't change, so can be a const at the module level, save re-evaluating (admittedly very very quick re-evaluation!)

Reply:
Done